--- origsrc/execjs-2.2.2/lib/execjs/runtimes.rb	2014-11-20 18:28:33.027660400 -0600
+++ src/execjs-2.2.2/lib/execjs/runtimes.rb	2014-11-20 18:29:38.938030000 -0600
@@ -27,7 +27,7 @@ module ExecJS
 
     JavaScriptCore = ExternalRuntime.new(
       name:        "JavaScriptCore",
-      command:     "/System/Library/Frameworks/JavaScriptCore.framework/Versions/A/Resources/jsc",
+      command:     "jsc-3",
       runner_path: ExecJS.root + "/support/jsc_runner.js"
     )
 
